examples/go-xdp-counter/main.go

```diff
@@ -48,15 +48,37 @@ func main() {
 		return
 	}
 
-	var mapPath string
+	var statsMap *ebpf.Map
+
+	opts := &ebpf.LoadPinOptions{
+		ReadOnly:  false,
+		WriteOnly: false,
+		Flags:     0,
+	}
+
+	c := bpfdHelpers.GetClientOrDie()
```

Comment:
Same comments from the go-tc-counter example:
This needs to only be called if running on Kubernetes.